### PR TITLE
Added command_rollershutter component

### PR DESCRIPTION
**Pull request by t30**

### CODE REVIEW

File: source/_components/rollershutter.command.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Command line Rollershutter"
+description: "Instructions how to have rollershutters call command line commands."
+date: 2016-02-22 13:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: command_line.png
```
Comment (balloob) [inline source/_components/rollershutter.command.markdown]: Can you please remove this to stay consistent with the other command line platforms.

Comment (t30) [inline source/_components/rollershutter.command.markdown]: it is my intention, if you agree, to add the same icon to all the command line platforms.

Comment (balloob) [inline source/_components/rollershutter.command.markdown]: Not a fan.

For generic components we use material design icons from
https://materialdesignicons.com/

On Mon, Feb 22, 2016 at 1:46 PM, Matteo Lampugnani <notifications@github.com
> wrote:

> In source/_components/rollershutter.command.markdown
> <https://github.com/balloob/home-assistant.io/pull/244#discussion_r53698421>
> :
>
> > @@ -0,0 +1,38 @@
> > +---
> > +layout: page
> > +title: "Command line Rollershutter"
> > +description: "Instructions how to have rollershutters call command line commands."
> > +date: 2016-02-22 13:45
> > +sidebar: true
> > +comments: false
> > +sharing: true
> > +footer: true
> > +logo: command_line.png
>
> it is my intention, if you agree, to add the same icon to all the command
> line platforms.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant.io/pull/244/files#r53698421>.
>



-- 
PaulusSchoutsen.nl
It's nice to be important but it's more important to be nice.

**Comment by GreenTurtwig** [comment]
I added `command_line.png` (MaterialDesignIcons.com console icon) to all the command line components so it should be fine. :-)

**Pull request merged**